subsys/mgmt/mcumgr/grp/stat_mgmt/Kconfig

```diff
@@ -32,6 +32,14 @@ config MCUMGR_GRP_STAT_MAX_NAME_LEN
 	  stat read commands.  If a stat group's name exceeds this limit, it will
 	  be impossible to retrieve its values with a stat show command.
 
+config MCUMGR_STAT_MAX_NAME_LEN
```

Comment:
Must follow naming rule from top of file:
```
# Options defined in this file should be prefixed:
#  MCUMGR_GRP_STAT_ -- general group options;
```

Reply:
I changed it to reuse the existing config.